packages/transaction_signing_gateway/lib/model/transaction_broadcasting_failure.dart

```diff
@@ -0,0 +1,5 @@
+abstract class TransactionBroadcastingFailure {}
```

Comment:
we'd need a `type` here, if possible. similar to what we have in `TransactionSigningFailure` in order to allow end users to `switch` over possible broadcasting errors.

Reply:
✅